### PR TITLE
Add CSS values test for mixed units in calc & media query

### PR DESCRIPTION
**Pull request by muan**
This is a test for a bug in Chrome [as reported](https://bugs.chromium.org/p/chromium/issues/detail?id=843584).

This works in Firefox and Safari.

|Chrome|Firefox|Safari|
|-|-|-|
|Chrome: an orange bar is shown|Firefox: a blue bar is shown|Safari: a blue bar is shown|

### CODE REVIEW

File: css/css-values/calc-mixed-units.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<html lang="en">
```
Comment (dbaron) [inline css/css-values/calc-mixed-units.html:1]: It's probably better to test in standards mode (i.e., with `<!DOCTYPE HTML>`) unless there's a good reason to test in quirks mode.

File: css/css-values/calc-mixed-units.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  <head>
+    <style>
+      .box {
+        background: rgb(0, 0, 255);
+        width: 100vw;
+        height: 10px;
+      }
+
+      @media (min-width: calc(100vw - 1em)) {
```
Comment (dbaron) [inline css/css-values/calc-mixed-units.html:10]: I think you should be able to come up with a much stricter test here.  If you put a test inside an iframe with a known size, and assume the default `16px` font size, you should be able to test queries for `width` and `height` exactly without using either `min-` or `max-`.  A few of those (different combinations of `vw`, `em`, and `px`, at least) would seem to be a somewhat comprehensive test for "calc-mixed-units", whereas passing this test alone doesn't ensure very much.

**Review by dbaron — CHANGES_REQUESTED** [review]
I think it should be possible to make this test quite a bit stronger; see below.

**Comment by muan** [comment]
@dbaron Thanks for the review. I've updated the test to include a couple unit pairings, and there are now test failures around `vh` in Safari as well.

Chrome with 6 failed assertions, Firefox with 0 failed assertions, Safari with 3 failed assertions

File: css/css-values/calc-mixed-units.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>test calc with mixed units</title>
```
Comment (dbaron) [inline css/css-values/calc-mixed-units.html:4]: The title should probably mention that media queries are involved.

**Review by dbaron — APPROVED** [review]
These look great.  I just noticed (oops, I hit the "Add Single Comment" button instead of the "Start Review" button) that the test doesn't mention media queries at all, though, which it probably should, at least in the title.

I haven't yet looked at why some of the checks are failing...

**Comment by muan** [comment]
Thanks again! I edited the title and renamed the file so it goes with the rest of `calc-in-media-queries-*.html`.

**Review by dbaron — APPROVED** [review]

**Pull request merged**